cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -705,11 +738,15 @@ notifyBlockForging fStats tr = Tracer $ \case
             (\fts -> (fts { ftsNodeIsLeaderNum = ftsNodeIsLeaderNum fts + 1
                           , ftsLastSlot = slot },
                       ftsNodeIsLeaderNum fts + 1))
-  Consensus.TraceForgedBlock {} ->
+  Consensus.TraceForgedBlock {} -> do
     traceCounter "blocksForgedNum" tr
       =<< mapForgingCurrentThreadStats fStats
             (\fts -> (fts { ftsBlocksForgedNum = ftsBlocksForgedNum fts + 1 },
                        ftsBlocksForgedNum fts + 1))
+    meta <- mkLOMeta Critical Public
+    let tBlocksUncoupled = fsBlocksUncoupled fStats
+    traceI tr meta "mySelectedBlocks" =<< STM.modifyReadTVarIO tBlocksUncoupled (+ 1)
```

Comment:
`mySelectedBlocks` sounds like the number of blocks I have forged that _are_ selected. But the metric is the number of such blocks that _are not_ selected. Just sharing my reaction, FWIW.

Reply:
Oops.  The string was not updated.  Will update it to `blocksUncoupled` - the number of such blocks that are not selected.

Reply:
Fixed!